wdl-analysis/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+
+* `missing_call_input` now gives a warning for missing inputs when `allowNestedInputs:true` ([#344]https://github.com/stjude-rust-labs/wdl/pull/344).
```

Comment:
While this is accurate, I had to read the associated issue to understand the behavior change, so this may need some rewording to be clear to users.

Reply:
@Pabsthegreat please reword this entry. And move it to `Added` instead of `Changed`. I think a key piece is that this did not change the `Error` behavior, it just adds a `Warning` case.